app/presenters/graph_presenter.rb

```diff
@@ -1,4 +1,6 @@
 class GraphPresenter
+  delegate :title, to: :start_node_presenter
```

Comment:
Nit: The bug fix for the title and moving the graph presenter to correct location should be a separate commits (or even PRs) - seems out of scope for the proposed changes.

Reply:
I'll move to a separate PR